app.js

```diff
@@ -2,6 +2,11 @@ const Emitter = require('events');
 const assert = require('assert');
 const transform = require('sdp-transform');
 const { v4: uuidv4 } = require('uuid');
+var BoundaryTag="--uniqueBoundary"
+
+if (process.env.JAMBONES_SIPREC_TYPE=="SMART_TAP"){
```

Comment:
since there may be other "types" in the future, please just make a switch statement with a case for 'SMART_TAP' and a default for everything else